src/components/Banner/hooks.tsx

```diff
@@ -58,7 +58,7 @@ export const useAlertBanner = (
       ignoredBannerMessages.current.add(props.message);
       setBannerProps((previousBannerProps) => [...previousBannerProps, props]);
     },
-    [bannerProps, ignoredBannerMessages.current]
```

Comment:
Tiny fix! I didn't end up using it in this PR, but this callback (`showAlert`) had an unused dependency on `bannerProps` that made the callback update more often than it needed to.